lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_extract_keys_from_strings_files.rb

```diff
@@ -45,10 +45,10 @@ def self.run(params)
       # @return [Hash<String, Array<String>>] The hash listing the keys to extract for each target file
       #
       def self.keys_list_per_target_file(original_files)
-        original_files.map do |original_file|
+        original_files.to_h do |original_file|
           keys = Fastlane::Helper::Ios::L10nHelper.read_strings_file_as_hash(path: original_file).keys
           [original_file, keys]
-        end.to_h
+        end
```

Comment:
Same question here as https://github.com/wordpress-mobile/release-toolkit/pull/464/files#r1174292121

Reply:
Same, `original_files` seems to be an `Enumerable` to me there so we should be ok.

Reply:
Thank you for double-checking my assumption 👍